cookiecutter.json

```diff
@@ -16,8 +16,10 @@
     "Amazon SES",
     "Custom SMTP"
   ],
+  "use_graphql": "n",
   "client_app": [
     "Vue3",
+    "React",
     "None"
```

Comment:
If you choose React, do you also NEED graphql? (or vice versa?) Or does our React setup work without it?
Do we have (or are planning) support for graphql in Vue?

Reply:
We don't NEED graphql if we're using React, but it does leave the API fetching layer up to the dev who generates the project. It could pretty easily be integrated with DRF on the backend.